app/models/usage_report.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2014 - 2023 UNICEF. All rights reserved.
+
+# class for Usage Report
+class UsageReport
+  include ActiveModel::Model
+
+  class << self
```

Comment:
Since the point of this is to run the `.count` queries, move the counts to this class